cypress.json

```diff
@@ -1,9 +1,11 @@
 {
   "baseUrl": "http://localhost:8080",
+  "defaultCommandTimeout": 7000,
   "fixturesFolder": "tests/e2e/fixtures",
-  "integrationFolder": "tests/e2e/specs",
+  "integrationFolder": "tests/e2e/cucumber",
   "pluginsFile": "tests/e2e/plugins/index.js",
   "screenshotsFolder": "tests/e2e/screenshots",
   "supportFile": "tests/e2e/support/index.ts",
-  "video": false
+  "video": false,
+  "testFiles": "**/*.feature"
```

Comment:
https://github.com/TheBrainFamily/cypress-cucumber-preprocessor#bundled-features-files

_When running Cypress tests in a headless mode, the execution time can get pretty bloated, this happens because by default Cypress will relaunch the browser between every feature file. The cypress-cucumber-preprocessor gives you the option to bundle all feature files before running the tests, therefore reducing the execution time.

You can take advantage of this by creating .features files. You choose to have only one in the root of the directory cypress/integrations or per directory._